src/components/Shared/Header/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { get } from 'lodash';
+
+import CtxFloodsLogoDarkSvg from 'images/ctx-floods-logo-dark.svg';
+import './Header.css';
+
+export default class Header extends Component {
+  componentDidMount() {
+    const pathname = get(this, 'props.location.pathname');
+    const suffix = pathname ? `- ${pathname}` : '';
+    document.title = `CTXFloods ${suffix}`;
+  }
+
+  render() {
+    return (
+      <div className="Header">
+        <div className="Header__main">
+          <h1 className="Header__logo">
+            <img
+              src={CtxFloodsLogoDarkSvg}
+              alt="CTXfloods | Central Texas Floods"
+            />
+            <div className="Header__title">{this.props.title}</div>
```

Comment:
should we require a title with proptypes here?

Reply:
Added the prop, but its optional. The regular pages don't have a title:

![image](https://user-images.githubusercontent.com/1275831/38632156-7650543c-3d81-11e8-94dd-113cb7c70c0d.png)